### PR TITLE
docs: add entry for uploading new version to ppa

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
docs: add entry for uploading new version to ppa

Add documentation informing how we should upload new uaclient packages to our maintained ppas

## Test Steps
Verify if this guide is enough to allow new users to upload new uaclient packages to the ppa

## Desired commit type::

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -61,3 +61,56 @@ Manually perform a binary package copy from Daily PPA to Premium PPA and notify
...
+      * This will generate a list of commits that could be included in the changelog. If you don't
+        have `log2dch`, you can get it from the [uss-tableflip](https://github.com/canonical/uss-tableflip)
+      * You don't need to include all of the commits generated. Remember that the changelog should
+        be read by the user to understand the new features/modifications in the package. If you
+        think a commit will not add that much to the user experience, you can drop it from the
+        changelog
+      * To structure the changelog you can use the other entries as example. But we basically try
+        keep this order: debian changes, new features/modifications, testing
+ 3. Start building the package:
+    * *WARNING* Build the package in a clean environment. The reason for that is because the package
```
Comment (blackboxsw) [inline RELEASES.md:88]: Good note on this to avoid our previous mistakes.

File: RELEASES.md [hunk 2]
```diff
@@ -61,3 +61,56 @@ Manually perform a binary package copy from Daily PPA to Premium PPA and notify
...
+      will contain everything that it is present in the folder. If you are storing credentials or
+      other sensible development information in your folder, they will be upload too when we send
```
Comment (blackboxsw) [inline RELEASES.md:90]:
```suggestion
      other sensible development information in your folder, they will be uploaded too when we send
```

File: RELEASES.md [hunk 3]
```diff
@@ -61,3 +61,56 @@ Manually perform a binary package copy from Daily PPA to Premium PPA and notify
...
+      the package to the ppa. A clean environment is the safest way to perform this.
+    * Build the necessary artifacts that allow building the package
+      * Guarantee that you have a gpg key in the system. You will use that gpg key to sign the
+        package.
+      * We can achieve that by running the `build-package` command. This script is also found on the
+        [uss-tableflip](https://github.com/canonical/uss-tableflip) repo.
+      * This script will generate all the package artifacts in a folder named `out`. This will be a
+        top level folder than the one you are running `build-package` command.
+    * Verify if we can build the package:
```
Comment (blackboxsw) [inline RELEASES.md:99]:
```suggestion
      * This script will generate all the package artifacts in the parent directory as `../out`.
    * Verify if we can build the package:
```

File: RELEASES.md [hunk 4]
```diff
@@ -61,3 +61,56 @@ Manually perform a binary package copy from Daily PPA to Premium PPA and notify
...
+        keep this order: debian changes, new features/modifications, testing
+ 3. Start building the package:
+    * *WARNING* Build the package in a clean environment. The reason for that is because the package
+      will contain everything that it is present in the folder. If you are storing credentials or
+      other sensible development information in your folder, they will be upload too when we send
+      the package to the ppa. A clean environment is the safest way to perform this.
+    * Build the necessary artifacts that allow building the package
+      * Guarantee that you have a gpg key in the system. You will use that gpg key to sign the
+        package.
+      * We can achieve that by running the `build-package` command. This script is also found on the
```
Comment (blackboxsw) [inline RELEASES.md:95]:
```suggestion
      * We can achieve that by running the `build-package` command when your current working is the top of source tree. This script is also found on the
```

**Review by blackboxsw — APPROVED** [review]
minor thoughts

File: RELEASES.md (hunk 3 above)
Comment (lucasmoura) [reply RELEASES.md:99]: Much better :)

**Comment by lucasmoura** [comment]
@blackboxsw updated

**Pull request merged**